File: crates/laya-parse/tests/fixtures/Sample.cs
using System;
using System.Collections.Generic;

namespace Example.Store
{
    /// <summary>A keyed store.</summary>
    public class KeyStore
    {
        private readonly Dictionary<string, string> _map = new();

        public int Count => _map.Count;

        public KeyStore()
        {
            _map["a"] = "b";
        }

        public string Get(string key)
        {
            return _map.TryGetValue(key, out var v) ? v : "";
        }
    }

    public interface IKeyed
    {
        string Key();
    }
}

File: crates/laya-parse/tests/fixtures/Sample.java
package com.example;

import java.util.HashMap;
import java.util.Map;

/** A simple store. */
public class Sample {
    private final Map<String, String> map = new HashMap<>();

    public Sample() {
        map.put("a", "b");
    }

    /** Gets a value. */
    public String get(String key) {
        String v = map.get(key);
        if (v == null) {
            return "";
        }
        return v;
    }

    @Override
    public String toString() {
        return map.toString();
    }
}

interface Keyed {
    String key();
}
